homeassistant/components/proxmoxve/__init__.py

```diff
@@ -108,94 +110,110 @@ def build_client() -> ProxmoxAPI:
                 _LOGGER.warning(
                     "Invalid credentials for proxmox instance %s:%d", host, port
                 )
+                hass.data[PROXMOX_CLIENTS][host] = None
```

Comment:
Can you set this once before the try instead of on every failure?

Reply:
indeed... Don't know why I didn't just do that to begin with. Nice catch!